### ISSUE TITLE
How long do you train to get a good result?

### ISSUE THREAD

**Issue by NineBall9** [experiencor/keras-yolo3#129]
And what the total loss can get to?
I have trained the model for 10h, and the loss gets to 10, not good enough to get a good result. So, what is your loss?
